### PR TITLE
feat: Add keywords and searching for sls onboarding

### PR DESCRIPTION
**Pull request by zongUMR**
## What has been implemented?

Closes https://app.asana.com/0/1200011502754281/1200046739955657/f

## Steps to verify

- Run X
- ...

## Todos:

- [ ] Write tests
- [ ] Write / update documentation
- [ ] Run Prettier

### CODE REVIEW

File: src/cli/interactive-onboarding/cn.js [hunk 1]
```diff
@@ -82,7 +82,9 @@ const getTemplatesFromRegistry = async (sdk) => {
       let name = item.name;
 
       if (item['description-i18n'] && item['description-i18n']['zh-cn']) {
-        name = `${name} - ${item['description-i18n']['zh-cn']}`;
+        name = `${name} - ${item['description-i18n']['zh-cn']}${
+          item.keywords && `-${item.keywords}`
```
Comment (timqian) [inline src/cli/interactive-onboarding/cn.js:86]: Looks the keywords will be shown in the console too. Is it what we want? @ole3021 

@zongUMR Can you share the screenshot of how the output looks like when user execute`sls`/`sls registry` in the command line after this change?

Comment (zongUMR) [reply src/cli/interactive-onboarding/cn.js:86]: Screen Shot 2021-03-15 at 10 20 00

1. show `keywords` after `description`
2. can seach `keywords`

Comment (ole3021) [reply src/cli/interactive-onboarding/cn.js:86]: @zongUMR  we don't need to show the keywords in the CLI result. 

* We should keep the info as short as possible, and useful. >> we can update the description with more help info for user to know which language and framework that used in the template. 
* we just need to provide a way for user to filter templates by language which is listed in the keywords. 

cc: @timqian

Comment (zongUMR) [reply src/cli/interactive-onboarding/cn.js:86]: Ok, remove `keywords` on terminal

**Review by timqian — APPROVED** [review]

**Pull request merged**